frontends/web/src/app.tsx

```diff
@@ -109,10 +113,14 @@ class App extends Component<Props, State> {
                 }
             }
         });
+        this.subscribtions.push(accounts(accounts => {
```

Comment:
I find it unclear to understand what's happening. Consider splitting in 2 steps:

```js
const unsub = accounts(list => { ...});
this.subscriptions.push(unsub);
```

Reply:
How about renaming `accounts()` to `syncAccounts()` ?

Reply:
how about
```js
this.unsubscribeList.push(subscribeAccountList(accounts => {
```

Reply:
or `syncAccountList`